xml/favicons.xml

```diff
@@ -1852,11 +1853,14 @@
   <fingerprint pattern="^c7f62c79333a43b6cd3f24d9e9c69526$">
     <description>iSTAR Ultra</description>
     <example>c7f62c79333a43b6cd3f24d9e9c69526</example>
-    <param pos="0" name="os.vendor" value="iStar"/>
-    <param pos="0" name="os.product" value="Linux"/>
-    <param pos="0" name="hw.vendor" value="iStar"/>
+    <param pos="0" name="os.vendor" value="Software House"/>
```

Comment:
@hdm @pbarry25 Since the iSTAR Ultra fingerprint was recently added (#382) I wanted to check with you to see if you have any concerns with these changes.

Reply:
@pbarry25 Agreed the changes seem reasonable via a Slack conversation.